app/database_etl/etl_main.py

```diff
@@ -495,17 +542,24 @@ def main():
     # Create dictionary to store bridge tables
     bridge_tables_dict = create_bridge_tables(dashboard_source, multi_select_tables_dict)
 
+    # Add mapped variables to master dashboard source table
+    dashboard_source = add_mapped_variables(dashboard_source)
+
     # Create research source table based on a subset of dashboard source df columns
-    research_source_cols = list(airtable_fields_config['research'].values())
-    research_source_cols.insert(0, 'source_id')
+    research_source_cols = list(airtable_fields_config['research'].values()) + ['gbd_region', 'gbd_subregion',
```

Comment:
why don't we just put those last 5 values in `airtable_fields_config['research']`

Reply:
because those 5 variables are not being pulled from airtable, they are being manually created. the column names in airtable fields config are ones we are pulling from airtable.

Reply:
checks out, can u add a comment describing this